src/main/java/uk/gov/dwp/dataworks/controller/DataKeyController.java

```diff
@@ -22,15 +25,15 @@ public DataKeyController(DataKeyService dataKeyService) {
         this.dataKeyService = dataKeyService;
     }
 
-    @RequestMapping(value = "/", method = RequestMethod.GET)
+    @RequestMapping(value = "", method = RequestMethod.GET)
```

Comment:
This might be better as a POST. GET should return a specified resource, but here we are creating one instead, which implies POST.

Reply:
We discussed, both approaches are valid.